In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
from sklearn.preprocessing import MinMaxScaler


# Leitura de Dados e filtração de Dados

In [2]:
# comando read_csv para tal.
dados = pd.read_csv('../../dados/AIRPOL_data.csv', delimiter=";", header=0, decimal=',')
dados = dados.drop(columns=['Unnamed: 8', 'Unnamed: 9', 'Unnamed: 10', 'Unnamed: 11', 'Unnamed: 12', 'Unnamed: 13', 'Unnamed: 14', 'Unnamed: 15'])

print(dados.info)


# Filtrar dados
soutEuDados = dados[dados['Country'].isin(['Greece', 'Spain', 'Italy', 'Portugal'])]
soutEuDadosNumericos = soutEuDados.select_dtypes(include=[np.number])



<bound method DataFrame.info of        Country NUTS_Code Air_Pollutant Outcome  Affected_Population  \
0      Albania        AL           NO2  Asthma            2337443.0   
1      Albania        AL         PM2.5  Asthma             456076.0   
2      Albania       AL0           NO2  Asthma            2337443.0   
3      Albania       AL0         PM2.5  Asthma             456076.0   
4      Albania      AL03         PM2.5  Asthma             149508.0   
...        ...       ...           ...     ...                  ...   
49135   Turkey     TR822           NO2  Stroke             213615.0   
49136   Turkey     TRB12           NO2  Stroke             447268.0   
49137   Turkey     TRB14           NO2  Stroke              76591.0   
49138   Turkey     TRB21           NO2  Stroke             696344.0   
49139   Turkey     TRB22           NO2  Stroke             362047.0   

       Populated_Area[km2]  Air_Pollution_Average[ug/m3]  Value  
0                  11299.0                       

In [3]:

corr = soutEuDadosNumericos.corr()
print(corr['Value'])


Affected_Population             0.563330
Populated_Area[km2]             0.531761
Air_Pollution_Average[ug/m3]   -0.014951
Value                           1.000000
Name: Value, dtype: float64


# Conlusão

# Pequena Interpretação

- Affected_Population e Populated_area são modeadamente positivamente afetados,
 Ao seja, o significa que, em geral, quanto maior a população e area populada afetada, maior tende a ser o "Value".

Air_Pollution_Average[ug/m3]: **-0.014951**
- A poluição média atmosferica não é muito fraca (praticamente nula).
  
